forage-android/src/main/java/com/joinforage/forage/android/network/data/CapturePaymentRepository.kt

```diff
@@ -123,14 +120,16 @@ internal class CapturePaymentRepository(
             }
 
             if (attempt == MAX_ATTEMPTS) {
-                logger.debug("Max attempts reached. Returning last response")
+                logger.e("Max polling attempts reached for capture request of Payment $paymentRef")
                 return ForageApiResponse.Failure(listOf(ForageError(500, "unknown_server_error", "Unknown Server Error")))
             }
 
             attempt += 1
             delay(POLLING_INTERVAL_IN_MILLIS)
         }
 
+        logger.i("Polling for capture request response succeeded for Payment $paymentRef")
```

Comment:
include `contentId` in log? Also, worth including `paymentRef` in log tag / Context for easier DataDog search?